npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker/subdependency_version_resolver.rb

```diff
@@ -107,9 +108,12 @@ def run_yarn_updater(path, lockfile_name)
           sleep(rand(3.0..10.0)) && retry
         end
 
-        def run_npm_updater(path, lockfile_name)
+        def run_npm_updater(path, lockfile_name, lockfile_content)
           SharedHelpers.with_git_configured(credentials: credentials) do
             Dir.chdir(path) do
+              npm_version = Dependabot::NpmAndYarn::Helpers.npm_version(lockfile_content)
+              puts npm_version
```

Comment:
Is there anything more sophisticated we can/should do other than just `put`? 😄 Maybe we could introduce the first tiny step for actual logging support and set up a `Dependabot.logger`, it could just be:

```ruby
class Dependabot
  def self.logger
    @logger ||= Logger.new(STDOUT)
  end
end
```

And then we could `Dependabot.logger.info npm_version` here

Reply:
Yeah great shout!